primitive-types/impls/serde/src/serialize.rs

```diff
@@ -32,7 +33,8 @@ fn to_hex<'a>(v: &'a mut [u8], bytes: &[u8], skip_leading_zero: bool) -> &'a str
 		idx += 2;
 	}
 
-	::std::str::from_utf8(&v[0..idx]).expect("All characters are coming from CHARS")
+	// SAFETY: all characters come from CHARS array.
+	unsafe { std::str::from_utf8_unchecked(&v[0..idx]) }
```

Comment:
Is there a benchmark that proves this is faster? Without a benchmark it seems it's just guessing.

Reply:
If you look at the implementation: https://doc.rust-lang.org/beta/src/core/str/mod.rs.html#337-340
It clearly should be faster ;)

Reply:
Added u256 bench in #209

Reply:
@tomusdrw I don’t have a benchmark.  That said, this is a case where the use of `unsafe` is justified.